source/install/cloud-changelog.md

```diff
@@ -4,12 +4,60 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2023-06-29](#release-2023-06-29)
 - [Release 2023-06-13](#release-2023-06-13)
 - [Release 2023-05-31](#release-2023-05-31)
 - [Release 2023-05-01](#release-2023-05-01)
 - [Release 2023-04-21](#release-2023-04-21)
 - [Release 2023-03-29](#release-2023-03-29)
-- [Release 2023-03-20](#release-2023-03-20)
+
+## Release 2023-06-29
+
+### Highlights
+ - Insights has been disabled for all new instances and for existing servers that upgrade to v8.0. The Insights feature flag can be enabled with environment variables via `MM_FEATUREFLAGS_INSIGHTSENABLED=true`. See more details in [this forum post](https://forum.mattermost.com/t/proposal-to-revise-our-insights-feature-due-to-known-performance-issues/16212) on why Insights has been disabled.
+ - Boards has been moved from a product back to a plugin and is now disabled by default. ``EnablePublicSharedBoards`` Boards product setting was removed from the config and the **System Console**.
```

Comment:
@esethna Can you confirm if "and is now disabled by default" is for all instances or for new installs only? Thank you.

Reply:
@amyblais Was the monorepo shipped in a previous versions? I don't see that it was and the change log references boards and playbooks as "pre-packaged". I see the note in 7.10 about  "Playbooks and Boards will be core parts of the product that cannot be disabled." 

I am wondering for the change log if we adjust that sentence or add a note to the 8.0 that that previous decision was changed?

Reply:
That would mean that we could adjust the first part of this sentence.

Reply:
@wiersgallak Monorepo was shipped in April 21st cloud release https://docs.mattermost.com/install/cloud-changelog.html#release-2023-04-21.
I'm planning to adjust the wording for monorepo for the v8.0 changelog.

Reply:
@amyblais after confirming the behavior with Team ABC, added a suggestion below. 
```suggestion
 - Boards has been moved from part of the core product back to a plugin. If you previously had the plugin disabled, it will be disabled now, even if you enabled it as part of the core product. You can re-enable it from the **System Console > Plugin** settings.  The Boards configuration, ``EnablePublicSharedBoards``, was also removed from the config and the **System Console**.
```